common/analysis/BUILD

```diff
@@ -43,16 +44,32 @@ cc_library(
     ],
 )
 
+genlex(
+    name = "lint_waiver_config_lex",
+    src = "lint_waiver_config.lex",
+    out = "lint_waiver_config.yy.cc",
+)
+
 cc_library(
     name = "lint_waiver",
-    srcs = ["lint_waiver.cc"],
-    hdrs = ["lint_waiver.h"],
+    srcs = [
+        "lint_waiver.cc",
+        "lint_waiver_config.yy.cc",
+        "lint_waiver_config.cc",
```

Comment:
Recommend pulling these into own `cc_library` much like `verilog_lexer` in verilog/parser/BUILD, even if it is used only in one place today.  It makes it easier for some automatic BUILD maintenance tools we have to update/refactor things incrementally.